force-app/main/default/lwc/rd2EntryFormScheduleSection/rd2EntryFormScheduleSection.js

```diff
@@ -78,6 +89,24 @@ export default class rd2EntryFormScheduleSection extends LightningElement {
             this.setFields(this.rdObjectInfo.fields);
             this.buildFieldDescribes();
             this.isLoading = !this.isEverythingLoaded();
+            
+
+            this.cssHideExperienceSite = this.isExperienceSite ? 'slds-hide' : '';
+            this.cssHideOnlyPaymentModal = this.isPaymentModal ? 'slds-hide' : '';
+            this.cssHideOnlyAmountFrequencyModal = this.isAmountFrequencyModal ? 'slds-hide' : '';
+            
+            
+            if(this.isExperienceSite) {
```

Comment:
This block of code is unrelated to the response from this `@wire`call, can we call this in the `connectedCallback` or better yet the `renderedCallback` hook instead, also we will greatly benefit from putting this code block in its own named function so we can identify its use.  

A jest test should be added to validate the UI behaviour we are introducing here.

Reply:
Ok, thanks.